demo/src/demo.ts

```diff
@@ -218,7 +218,53 @@ async function main() {
     document
   )
   console.log('✅ Credential created!')
+  
+  console.log('🖍️ Stream update...')
+  let argSignCallBack: any = async ({ data }) => ({
+    signature: delegateTwoKeys.assertionMethod.sign(data),
+    keyType: delegateTwoKeys.assertionMethod.type,
+  })
+
+  let newContent: any = {
+    name: 'Alice Senna',
+    age: 23,
+    id: '123456789987654311',
+    gender: 'Female',
+    country: 'India',
+  }
+
```

Comment:
use Cord.Content.fromSchemaAndContent to transform the newContent into IContent before passing it to the update function. The schema and holder details should be fetched from the existing document